src/backend/base/langflow/graph/edge/schema.py

```diff
@@ -32,3 +34,55 @@ def format(self, sep: str = "\n") -> str:
                 for result_pair in self.result_pairs[:-1]
             ]
         )
+
+
+class TargetHandle(BaseModel):
+    fieldName: str = Field(..., description="Field name for the target handle.")
+    id: str = Field(..., description="Unique identifier for the target handle.")
+    inputTypes: Optional[List[str]] = Field(None, description="List of input types for the target handle.")
+    type: str = Field(..., description="Type of the target handle.")
+
+
+class SourceHandle(BaseModel):
+    baseClasses: list[str] = Field(default_factory=list, description="List of base classes for the source handle.")
+    dataType: str = Field(..., description="Data type for the source handle.")
```

Comment:
`data_type: str = Field(..., alias="dataType", description="Data type for the source handle.")`